app/src/main/java/org/jellyfin/mobile/player/PlaybackMenus.kt

```diff
@@ -94,25 +95,30 @@ class PlaybackMenus(
             joinToString(
                 "\n",
                 "${fragment.getString(R.string.playback_info_video_streams)}:\n",
-                limit = 3,
-                truncated = fragment.getString(R.string.playback_info_and_x_more, size - 3)
-            ) {
-                val bitrate = it.bitRate ?: 0
+                limit = MAX_VIDEO_STREAMS_DISPLAY,
+                truncated = fragment.getString(R.string.playback_info_and_x_more, size - MAX_VIDEO_STREAMS_DISPLAY)
+            ) { stream ->
+                val bitrate = stream.bitRate ?: 0
+
+                @Suppress("MagicNumber")
                 val bitrateString = when {
-                    bitrate > 1_000_000 -> String.format("%.2f Mbps", bitrate.toDouble() / 1_000_000)
-                    bitrate > 1_000 -> String.format("%.2f kbps", bitrate.toDouble() / 1_000)
-                    else -> String.format("%d kbps", bitrate / 1000)
+                    bitrate > 1_000_000 -> "%.2f Mbps".format(Locale.getDefault(), bitrate.toDouble() / 1_000_000)
+                    bitrate > 1_000 -> "%.2f kbps".format(Locale.getDefault(), bitrate.toDouble() / 1_000)
+                    else -> "%d kbps".format(bitrate / 1000)
                 }
-                "- ${it.displayTitle} ($bitrateString)"
+                "- ${stream.displayTitle} ($bitrateString)"
             }
         }
         val audioTracksInfo = mediaSource.audioStreams.run {
             joinToString(
                 "\n",
                 "${fragment.getString(R.string.playback_info_audio_streams)}:\n",
-                limit = 5,
-                truncated = fragment.getString(R.string.playback_info_and_x_more, size - 3)
```

Comment:
This was actually bug, should also be 5. Thanks detekt for forcing me to use a const :smile: